File: src/main/java/me/racci/sylphia/hook/perms/PermManager.java
package me.racci.sylphia.hook.perms;

import org.bukkit.entity.Player;

import java.util.List;

public interface PermManager {

	void addPermission(Player player, List<String> permissions);

	void addPermission(Player player, String permission);

	void removePermission(Player player, List<String> permissions);

	void removePermission(Player player, String permission);
}

File: src/main/kotlin/me/racci/sylphia/hook/perms/PermManager.kt
package me.racci.sylphia.hook.perms

import java.util.UUID

interface PermManager {
    fun addPermission(uuid: UUID, permissions: List<String>)
    fun addPermission(uuid: UUID, permission: String)
    fun removePermission(uuid: UUID, permissions: List<String>)
    fun removePermission(uuid: UUID, permission: String)
}